fixtures/mysql/edxapp.sql

```diff
@@ -13,9 +16,23 @@ CREATE TABLE IF NOT EXISTS auth_user (
     email NVARCHAR(255),
     username VARCHAR(255)
 );
-INSERT INTO courses_course (id, `key`) VALUES (1, 'course-v1:FUN-MOOC+00001+session01');
-INSERT INTO courses_course (id, `key`) VALUES (2, 'course-v1:FUN-MOOC+00002+session01');
-INSERT INTO student_courseaccessrole (id, `user_id`, `course_id`, role) VALUES (1, 1, 'course-v1:FUN-MOOC+00001+session01', 'staff');
-INSERT INTO student_courseaccessrole (id, `user_id`, `course_id`, role) VALUES (2, 1, 'course-v1:FUN-MOOC+00002+session01', 'instructor');
+CREATE TABLE IF NOT EXISTS student_courseenrollment (
+    id INT PRIMARY KEY NOT NULL,
+    `user_id` INT,
+    `course_id` VARCHAR(255),
+    `is_active` INT
+);
+INSERT INTO courses_course (id, `key`, `title`, `start_date`, `end_date`) VALUES (1, 'course-v1:FUN-MOOC+00001+session01', 'FUN Mooc Course 1 Session 1', DATE('2000-01-01'), DATE('2030-01-01'));
+INSERT INTO courses_course (id, `key`, `title`, `start_date`, `end_date`) VALUES (2, 'course-v1:FUN-MOOC+00002+session01', 'FUN Mooc Course 2 Session 2', DATE('2000-01-01'), DATE('2030-01-01'));
```

Comment:
Ideally, course dates should cover the same period as ES data, _i.e._ now - 4 weeks?